src/Propel/Generator/Command/MigrationStatusCommand.php

```diff
@@ -142,7 +142,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         } else {
             $output->writeln(sprintf('No migration file found in "%s".', $dir));
 
-            return false;
+            return 1;
```

Comment:
Get https://github.com/propelorm/Propel2/pull/1590/files#diff-982b6a0d858ab1ce8a351b2edcb63ba7R28-R29 merged first as that one uses proper class constants for this.

Reply:
Alternatively, you can merge relevant hunks and useful snippets from that PR into yours here.
That might be easier to get this PR here shipped.

Reply:
How is use of constants blocking this from getting merged?

Reply:
Technically, you could also fix it up after merge, true.
When you change this value though, it would make more sense to not use magic numbers but the constants right away.
But if it is easier we can also fix it up post merge then.